DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -48,58 +48,190 @@
+				<field type="[S]" arg="" name=""/>
+				<field type="[S]" arg="" name="Max Angle of Random Turns"/>
+				<field type="[S]" arg="" name=""/>
+			</Properties1>
+			<Fields1>
+				<Field name="" dataType="s32" isBool="0"/>
+				<Field name="" dataType="s32"/>
+				<Field name="" dataType="s32" isBool=""/>
+			</Fields1>
+		</ActionID>
+		<ActionID ID="128" name="FFX Duration">
+			<Properties1>
+				<field type="[S]" arg="" name="" wiki="How long this effect will spawn particles in seconds. If set to a negative value, it will never stop unless something else makes it stop."/>
+			</Properties1>
+			<Fields1>
+				<Field name="Spawner Delay" dataType="f32"/>
+				<field name="" dataType="s32" isBool=""/>
+				<Field name="" dataType="s32"/>
+				<Field name="" dataType="f32"/>
+			</Fields1>
+		</ActionID>
+		<ActionID ID="129" name="Particle Duration">
+			<Properties1>
+				<field type="[S]" arg="" name="" wiki="How long the particles last in seconds. If set to a negative value, the particles will last forever."/>
+			</Properties1>
+			<Fields1>
+				<Field name="" dataType="f32" wiki="This has a strange effect on effects using action 105's 5th property (normally controls how well particles follow the effect) and action 34's rotation speed properties. When this is set to 1 while those other action properties are also used, it makes the particles fly away from where they spawned very quickly for some reason. Setting it to 0 disables this strange effect. Need to do some more testing with this to figure out what exactly is going on"/>
+			</Fields1>
+		</ActionID>
+		<ActionID ID="131" name="Effect Multiplier (Scale, Color)">
+			<Properties1>
+				<field type="[S]" arg="" name="" wiki="On one effect, this seemed to control the vertical speed of the particles it spawned, but on another it seemed to not do anything, so this is probably linked to some other property or field somewhere in some way"/>
+				<field type="[S]" arg="[E]" name="X Scale"/>
+				<field type="[S]" arg="[E]" name="Y Scale"/>
+				<field type="[S]" arg="[E]" name="Z Scale"/>
+				<field type="[C]" arg="[E]" name="Color"/>
 			</Properties1>
-			<Properties2>
-			</Properties2>
 			<Fields1>
 				<Field name="Keep aspect ratio" dataType="s32" isBool="" wiki="When set to 0, you can scale the particles in each direction individually using the scale multipliers in this action's properties When set to 1, the X scale multiplier will apply to the Y and Z axes as well, so the particles will maintain their aspect ratio. I think the two(?) other scale multipliers do nothing when this is set to 1."/>
 			</Fields1>
-			<Fields2>
-			</Fields2>
 		</ActionID>
-		<ActionID ID="132" name="FFXReference()">
+		<ActionID ID="132" name="FFX Reference">
 			<Fields1>
-				<Field name="FFX ID" dataType="s32" />
+				<Field name="FFX ID" dataType="s32"/>
 			</Fields1>
 		</ActionID>
-		<ActionID ID="199" name="DisableEffect()">
+		<ActionID ID="199" name="Disable Action">
```

Comment:
Removing Function Style syntax in the name is a dangerous choice, however it doesn't serve much purpose to anyone who's not familiar to it, however i suggest giving action ID's visual cues that allow to easily identify them.